docs/source/markdown/podman-run.1.md

```diff
@@ -1528,6 +1534,13 @@ weight by **--blkio-weight-device** flag. Use the following command:
 $ podman run -it --blkio-weight-device "/dev/sda:200" ubuntu
 ```
 
+### Using a podman container from within a piped command
```

Comment:
I'm not entirely sure this title reflects the intention. I find it hard to parse. Maybe "Using a podman container *with input from a pipe*"? Or "Piping command output to a podman container"?